proto/substrait/algebra.proto

```diff
@@ -845,6 +848,11 @@ message Expression {
       int32 scale = 3;
     }
 
+    message Timestamp {
+      // the maximum precision is 9.
+      int32 precision = 1;
```

Comment:
Default value is likely 6 for backwards compatibility (although this is a completely different message type so I'm not sure that matters).

Reply:
If this is the type, there's no need, but it looked like this was replacing the value version of int timestamp.